web/src/components/people/AdminView.tsx

```diff
@@ -127,7 +127,10 @@ export default function AdminView() {
               <td className={styles.trainingInfo} data-cy="training">
                 <div className={styles.trainingRecord}>
                   {person.training_record.training_records?.map((training: TrainingRecord) => (
-                    <div key={`${training.kind}-${training.completed_at}`} className={styles.training}>
+                    <div
+                      key={`${person.user.id}-${training.kind}-${training.completed_at}`}
```

Comment:
I don't think this div key was unique enough(?)

Reply:
Your updated version looks more unique to me, so I'd go with that 👍